tools/hangar/__snapshots__/test_corpus/valid/bring_local_dir.test.w_compile_tf-aws.md

```diff
@@ -106,13 +106,13 @@ class $Root extends $stdlib.std.Resource {
     const w = $helpers.bringJs(`${__dirname}/preflight.widget-1.cjs`, $preflightTypesMap);
     const subdir = $helpers.bringJs(`${__dirname}/preflight.subdir2-6.cjs`, $preflightTypesMap);
     $helpers.nodeof(this).root.$preflightTypesMap = $preflightTypesMap;
-    const widget1 = new w.Widget(this, "widget1");
+    const widget1 = globalThis.$ClassFactory.new("rootpkg.subdir2.inner.Widget", w.Widget, this, "widget1");
```

Comment:
"rootpkg" is the default name we give for the assembly if the project has no name since every FQN needs one. It was an arbitrary choice (though I wanted to distinguish it from the word "root" which is used by itself in other parts of the SDK).